pkg/driver/driver.go

```diff
@@ -85,7 +85,7 @@ func (d Driver) OpenConnector(name string) (driver.Connector, error) {
 
 // Register makes our database Driver available under the name "icingadb-*sql".
 func Register(logger *logging.Logger) {
-	sql.Register(MySQL, &Driver{ctxDriver: &mysql.MySQLDriver{}, Logger: logger})
+	sql.Register(MySQL, &Driver{ctxDriver: &mySQLDriver{}, Logger: logger})
```

Comment:
Wait, we are already wrapping the driver. Why do you wrap it once more instead of just initializing the connection there?

Reply:
Sure, we have a common _Driver_, but my stuff is only for MySQL.

Reply:
Done.